contracts/staking/StakingPool.sol

```diff
@@ -253,7 +245,7 @@ contract StakingPool is
             rewardsTimestamp > info.epochStartTimestamp + info.epochDuration,
             "StakingPool: init rewards"
         );
-        require(info.treasury != address(0), "StakePool: nonzero treasury"); // TODO: are we checking if the treasury is whitelisted to that daoId
+        require(info.treasury != address(0), "StakePool: nonzero treasury");
```

Comment:
nit; s/nonzero/zero i.e. Message should be `zero`.

The require fails only when `.info.treasury` is address zero.

Reply:
good catch, blaming this one on github copilot